### PR TITLE
Implemented delete meeting permanently function

### PR DESCRIPTION
**Pull request by ThaminduChankana**

### CODE REVIEW

File: src/api/controllers/Meeting.controller.ts [hunk 1]
```diff
@@ -59,3 +59,20 @@ export const getInternalMeetingById = async (request: Request, response: Respons
 		request.handleResponse.errorRespond(response)("MeetingId not found");
 	}
 };
+
+export const deleteMeetingPermanently = async (request: Request, response: Response, next: NextFunction) => {
+	const meetingId = request.params.meetingId;
+	if (meetingId) {
+		await MeetingService.deleteMeetingPermanently(meetingId)
```
Comment (rusiruavb) [inline src/api/controllers/Meeting.controller.ts:66]: If you use `then` `catch`, no need to use `await` keyword.

**Review by miyurugunarathna — APPROVED** [review]
LGTM

File: src/api/services/Meeting.service.ts [hunk 2]
```diff
@@ -66,3 +66,17 @@ export const fetchMeetingById = async (meetingId: string) => {
 			throw new Error(error.message);
 		});
 };
+
+export const deleteMeetingPermanently = async (meetingId: string) => {
+	if (meetingId) {
+		return await MeetingModel.findByIdAndDelete(meetingId)
```
Comment (rusiruavb) [inline src/api/services/Meeting.service.ts:72]: Remove `await` if you use `then` `catch`

**Review by rusiruavb — CHANGES_REQUESTED** [review]

File: src/api/controllers/Meeting.controller.ts [hunk 3]
```diff
@@ -59,3 +59,20 @@ export const getInternalMeetingById = async (request: Request, response: Respons
 		request.handleResponse.errorRespond(response)("MeetingId not found");
 	}
 };
+
+export const deleteMeetingPermanently = async (request: Request, response: Response, next: NextFunction) => {
```
Comment (senuravihanjayadeva) [inline src/api/controllers/Meeting.controller.ts:63]: Remove async also

**Review by senuravihanjayadeva — CHANGES_REQUESTED** [review]

**Pull request merged**